src/box/lua/connpool.lua

```diff
@@ -94,11 +94,15 @@ local function is_candidate_checked(candidate)
            conn.state == 'closed'
 end
 
-local function connect_to_candidates(candidates)
+-- This method connects to all of the specified instances
+-- and returns the set of successfully connected ones.
+local function acquire_all_candidates(instance_names)
```

Comment:
For me the renaming of this function is not needed. Even in the comment you say that this function 'connects'.

Reply:
This renaming has been made because I've wanted `acquire_***` function names to be more consistent internally.